src/components/DownloadTrajectoryMenu/index.tsx

```diff
@@ -53,22 +53,33 @@ const DownloadTrajectoryMenu = ({
         downloadFile(simulariumFile.name);
     };
 
+    const isDisabled = () => {
+        return !fileIsLoaded() || isBuffering;
+    };
+
+    const getTooltipOffset = () => {
```

Comment:
similarly to above, this could just be

`const tooltipOffset = isDisabled ? [25, -30] : [40, -18];`